src/components/WalletSelectorField/WalletSelectorFieldTs.ts

```diff
@@ -81,7 +83,8 @@ export class WalletSelectorFieldTs extends Vue {
     if (!this.knownWallets || !this.knownWallets.length) {
       return []
     }
-
+    // force update the knownWallets
+    this.hasUpdatedCurrentWallet
```

Comment:
`this.hasUpdatedCurrentWallet` will just drop a boolean on the stack ; why did you add this line?

Reply:
> `this.hasUpdatedCurrentWallet` will just drop a boolean on the stack ; why did you add this line?
`this.hasUpdatedCurrentWallet` is reactive, the   currentWallets will  re-evaluate when the `this.hasUpdatedCurrentWallet` changes
[https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods](url)